GcmEndpoints/README.md

```diff
@@ -7,7 +7,9 @@ This backend template employs [Google Cloud Endpoints](https://developers.google
 
 # 1. Adding a backend in Android Studio
 
-To add the backend to your existing Android app from this backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
+This guide builds on top of the [GCM sample Quickstart app](https://github.com/googlesamples/google-services/tree/master/android/gcm). If you already have an existing application, then you can still reference the Quickstart app as a guide to setting the GCM client code.
+
+To add the backend to your existing Android app from the GCM backend template, open Android Studio ([installation instructions](https://developer.android.com/sdk/installing/studio.html)) and navigate to "File &rarr; New Module..." or right-click on your project and choose "New &rarr; Module".
```

Comment:
do we really need to link to android studio installation instructions here? I think we can count on developers figuring that out. Or  just put the link on "Android Studio" instead of separate words.